event_bus.go

```diff
@@ -151,12 +152,13 @@ func (bus *Bus) genReplyQueue() (string, error) {
 }
 
 func (bus *Bus) createTopic(topic string) error {
-	u, err := url.Parse(NSQ_URL)
+	s := strings.Split(NSQ_URL, ":")
+	port, err := strconv.Atoi(s[1])
 	if err != nil {
 		return err
 	}
 
-	uri := "http://" + u.Host + ":4151/topic/create?topic=" + topic
+	uri := "http://" + s[0] + ":" + strconv.Itoa(port+1) + "/topic/create?topic=" + topic
```

Comment:
@aalvesjr given the convention most used is to map tcp-port=4150 and http-port=4151 I increment 1 so we get the nsqd current node port